lib/vmdb/appliance.rb

```diff
@@ -146,24 +146,40 @@ def self.get_network
       retVal
     end
 
-    def self.init_diagnostics
-      log_dir = "/var/www/miq/vmdb/log"
-      gem_log = File.join(log_dir, "gem_list.txt")
-      rpm_log = File.join(log_dir, "package_list_rpm.txt")
+    def self.installed_rpms
+      File.open(log_dir.join("package_list_rpm.txt"), "a") do |file|
+        file.puts "start: date time is: #{Time.now.utc}"
+        LinuxAdmin.Rpm.list_installed.sort do |name, version|
```

Comment:
To @carbonin 's comment (which got hidden), you can't pass a block to sort like this.  This code will fail.  You need to tack on a .each: `LinuxAdmin.Rpm.list_installed.sort.each do |name, version|`